keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -341,68 +419,134 @@ Contains a unique identifier for the containing cluster.
 
 *   The identifier **must** either:
     *   be a valid [RFC-1123](https://tools.ietf.org/html/rfc1123) DNS label,
-    *   or be composed of two valid [RFC-1123](https://tools.ietf.org/html/rfc1123) DNS labels separated with a dot.
+    *   or be composed of two valid
+        [RFC-1123](https://tools.ietf.org/html/rfc1123) DNS labels separated
+        with a dot.
 *   The identifier **may** be used as a component in MCS DNS.
 *   The identifier **may** be a human readable description of its cluster.
 
 
 ##### Consumers
 
-*   **Must** be able to rely on the identifier existing, unmodified for the entire duration of its membership in a ClusterSet.
-*   **Should** watch the `cluster.clusterset.k8s.io` property to handle potential changes if they live beyond the ClusterSet membership.
-*   **May** rely on the existence of an identifier for clusters that do not belong to a ClusterSet so long as the implementation provides one.
+*   **Must** be able to rely on the identifier existing, unmodified for the
+    entire duration of its membership in a ClusterSet.
+*   **Should** watch the `cluster.clusterset.k8s.io` property to handle
+    potential changes if they live beyond the ClusterSet membership.
+*   **May** rely on the existence of an identifier for clusters that do not
```

Comment:
I think we can strike this one, too

Reply:
I think this has non-zero value - the statement is that it's not against the spirit of the API to use the cluster identifier outside of a clusterset context. I think that opens enough interesting use cases that it's worth keeping

Reply:
OK.  Not worth arguing too much.  It seems pointless but I don't care much :)